linked_list/linked_list.py

```diff
@@ -17,19 +17,61 @@ def get_next(self):
   def set_next(self, new_next):
     self.next_node = new_next
 
+
+
 class LinkedList:
   def __init__(self):
     self.head = None
     self.tail = None
+  
+  def get_length(self):
+    count = 0
+    counter = self.head
+    while counter:
+      count += 1
+      counter = counter.get_next()
+    return count    
 
   def add_to_tail(self, value):
-    pass
+    # self.tail.next_node = self.tail
+    # self.tail = Node(value)
+
+    self.tail = Node(value)
+    if self.head is None:
+      self.head = Node(value)
+      return
+    lastNode = self.head
+    while(lastNode.next_node):
+      lastNode = lastNode.next_node
+    lastNode.next_node = Node(value)
 
   def remove_head(self):
-    pass
+    if self.head == None:
```

Comment:
I would pick either `=` or `is` when checking for `None` and use it throughout. `is` is the preferred way https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_none.html